src/base/bittorrent/sessionimpl.cpp

```diff
@@ -2069,7 +2069,7 @@ lt::settings_pack SessionImpl::loadLTSettings() const
     // UPnP lease duration
     settingsPack.set_int(lt::settings_pack::upnp_lease_duration, UPnPLeaseDuration());
     // Type of service
-    settingsPack.set_int(lt::settings_pack::peer_tos, peerToS());
+    settingsPack.set_int(lt::settings_pack::peer_dscp, peerDSCP());
```

Comment:
Did I understand correctly that `lt::settings_pack::peer_dscp` was not just added, but already exists before that, so this PR will not break compatibility with previous (supported) libtorrent releases?